exchange/auction.go

```diff
@@ -244,10 +289,10 @@ func defTTL(bidType openrtb_ext.BidType, defaultTTLs *config.DefaultTTLs) (ttl i
 }
 
 type auction struct {
-	// winningBids is a map from imp.id to the highest overall CPM bid in that imp.
-	winningBids map[string]*pbsOrtbBid
-	// winningBidsByBidder stores the highest bid on each imp by each bidder.
-	winningBidsByBidder map[string]map[openrtb_ext.BidderName]*pbsOrtbBid
+	// We'll store the hightest bid comming from each Imp in the request
```

Comment:
Please correct the spelling errors (hightest, comming) and add that these maps map **imp IDs** to what they're mapped to (highest bid for the specified imp in first case, a map of the highest bid from each bidder for the specified imp in the second case).  Bids don't _come from_ the imps; they are _for_ the imps.